src/pantab/writer.cpp

```diff
@@ -540,7 +541,8 @@ void write_to_hyper(
   }
 
   const std::unordered_map<std::string, std::string> params = {
-      {"log_config", ""}};
+      {"log_config", ""}, {"default_database_version", std::to_string(version)}};
```

Comment:
Are there any other configuration items worth passing through? The other design consideration is if we want to explicitly enumerate these in our API (which puts some onus on us to manage backwards compatability) or if we would rather just accept a dictionary argument for `hyper_params`

Reply:
I like the way of letting the user pass a dictionary of parameters, as each person may want to use it differently, there are other parameters in the hyperprocess, such as "use_tcp_port"

Reply:
Sounds good. Let's try that here then. I don't know that the Hyper C++ API can handle complete forwarding of the dict, but we can at least unpack the ones we know about in the extension layer via `nb::try_cast` and warn for any others we cannot process